javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/IndyBootstrapDispatcher.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap;
+
+import java.lang.invoke.CallSite;
+import java.lang.invoke.ConstantCallSite;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+import java.lang.reflect.Array;
+
+/**
+ * Contains the bootstrap method for initializing invokedynamic callsites which are added via agent
+ * instrumentation.
+ */
+@SuppressWarnings("unused")
+public class IndyBootstrapDispatcher {
+
+  private static MethodHandle bootstrap;
```

Comment:
Can you add a comment explaining why a non-volatile variable is fine?

Reply:
TBH I'm not sure whether a non-volatile is actually fine here 😅 
This code was just taken from the elastic-apm-agent without reconsidering the missing `volatile`.

I think it should be volatile, it just has never caused any problems yet because the update of the variable happens at the very start of the agent, long time before the bootstrap is actually called.

Reply:
Hmm, that's even more the reason to either add a comment or `volatile` 😄